app.go

```diff
@@ -210,6 +210,13 @@ func StopTimeout(v time.Duration) Option {
 	})
 }
 
+// Visualizer enables providing visualization of the container dependency graph.
+func Visualizer() Option {
+	return optionFunc(func(app *App) {
```

Comment:
Why make this opt-in? There's no harm in putting it in by default.

Reply:
I didn't want to alter the default behavior.

Reply:
Discussed offline: We'll do this by default.